src/main/kotlin/org/jitsi_modified/impl/neomedia/rtp/sendsidebandwidthestimation/config/SendSideBandwidthEstimationConfig.kt

```diff
@@ -16,164 +16,101 @@
-             * (expressed as a proportion of lost packets).
-             *
-             * First, we'll check for an experimental value in the old config file,
-             * then we'll check for an experimental value in the new config file,
-             * then we'll fall back to the default value
-             */
-            class ExperimentalHighLossThresholdProperty : LegacyFallbackConfigProperty<Double>(
-                Double::class,
-                legacyName = "$LEGACY_BASE_NAME.highLossThreshold",
-                newName = "jmt.bwe.send-side.loss-experiment.high-loss-threshold",
-                readOnce = true
-            )
-            private val experimentalHighLossThresholdProp =
-                ExperimentalHighLossThresholdProperty()
-
-            @JvmStatic
-            fun experimentalHighLossThreshold(): Double = experimentalHighLossThresholdProp.value
-
-            /**
-             * The property that specifies the bitrate threshold when the
-             * loss probability experiment is active.
-             */
-            class ExperimentalBitrateThresholdProperty : FallbackProperty<Bandwidth>(
-                legacyConfigAttributes {
-                    name("$LEGACY_BASE_NAME.bitrateThresholdKbps")
-                    readOnce()
-                },
-                newConfigAttributes {
-                    name("jmt.bwe.send-side.loss-experiment.bitrate-threshold")
-                    readOnce()
-                    retrievedAs<String>() convertedBy(Bandwidth.Companion::fromString)
-                }
-            )
-            private val experimentalBitrateThresholdProp =
-                ExperimentalBitrateThresholdProperty()
-
-            @JvmStatic
-            fun experimentalBitrateThresholdKbps(): Int = experimentalBitrateThresholdProp.value.kbps.toInt()
-
-            /**
-             * The property that specifies the probability of enabling the
-             * timeout experiment.
-             */
-            class TimeoutExperimentProbabilityProperty : LegacyFallbackConfigProperty<Double>(
-                Double::class,
-                readOnce = true,
-                legacyName = "$LEGACY_BASE_NAME.timeoutExperimentProbability",
-                newName = "jmt.bwe.send-side.timeout-experiment.probability"
-            )
-
-            private val timeoutExperimentProbabilityProp =
-                TimeoutExperimentProbabilityProperty()
-
-            @JvmStatic
-            fun timeoutExperimentProbability(): Double = timeoutExperimentProbabilityProp.value
+    companion object {
+        private val defaultLowLossThreshold: Double by
+            config("jmt.bwe.send-side.low-loss-threshold".from(JitsiConfig.newConfig))
+
+        /**
+         * The low-loss threshold (expressed as a proportion of lost packets) when the loss probability
+         * experiment is *not* active
+         */
+        @JvmStatic
+        fun defaultLowLossThreshold() = defaultLowLossThreshold
```

Comment:
Why do we need both the `val` and `fun`? To make it nicer to access from java?

Reply:
Yeah, unfortunately I can't mark the prop fields as `JvmStatic`/`JvmField` since they use a delegate and the getXXX syntax was annoying in some places (especially with booleans) so  I went this route.